src/nodes/typeGuards/alias.ts

```diff
@@ -0,0 +1,3 @@
+import { isBlock } from "./simple";
+
+export const isFunctionBody = isBlock;
```

Comment:
Ehhhhhh, I'd forgotten / not come across this particular one before. Seems kinda useless. Is there a strong reason to add it? If not, I'd vote to not.

Reply:
TypeScript declares `export type FunctionBody = Block`.

This type guard probably isn't needed as `Block` can just be used anywhere `FunctionBody` is.
IntelliSense also seems to just use `Block` instead of `FunctionBody`.

I included simply for completeness.